src/api/index.js

```diff
@@ -0,0 +1,28 @@
+import { api } from '../lib/axios'
+
+export const getToken = async () => {
+  try {
+    const resp = await api.get('/auth/anonymous?platform=subscriptions')
+    return resp.data.token
+  } catch (e) {
+    return null
```

Comment:
FYI: maybe it's worth to add showing the error in console? (for better understanding the problem when it appears)
Users never check the console, but the developer who uses your app may be happier)

Reply:
it is quite interesting question, in one hand it improves debugging. But on another hand it could worsen lighthouse metrics and make some misunderstanding for the clients who knows about dev tools.

I will send this question to our mentor

Thank you for the comment